statistics/analyze_jobs_test.go

```diff
@@ -15,14 +15,13 @@
 package statistics
 
 import (
-	. "github.com/pingcap/check"
-)
-
-var _ = SerialSuites(&testStatisticsSerialSuite{})
+	"testing"
 
-type testStatisticsSerialSuite struct{}
+	"github.com/stretchr/testify/require"
+)
 
-func (s *testStatisticsSerialSuite) TestMoveToHistory(c *C) {
+func TestMoveToHistory(t *testing.T) {
+	t.Parallel()
 	ClearHistoryJobs()
```

Comment:
@feitian124 This method seems touch global state and the original test run in serial. Could you elaborate the reason change the behavior?

Reply:
sorry, my fault, should run in serial.
updated.